pkg/upgrade/upgrade.go

```diff
@@ -25,13 +25,14 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/opendatahub-io/opendatahub-operator/v2/apis/components"
```

Comment:
It looks like this is also imported on L29 as componentsold, are both intentional?

Reply:
There is slight difference `v2/apis/components` -- `v2/components`. Kept the package names same as the `componentsold` will be removed later

Reply:
I need to get my eyes checked :see_no_evil: